packages/tools/examples/crossHairs/index.ts

```diff
@@ -190,6 +199,28 @@ addDropdownToToolbar({
   },
 });
 
+addToggleButtonToToolbar({
+  id: 'syncSlabThickness',
+  title: 'Sync Slab Thickness',
+  defaultToggle: false,
+  onClick: (toggle) => {
+    synchronizer.setEnabled(toggle);
+  },
+});
+
+function setUpSynchronizers() {
+  synchronizer = createSlabThicknessSynchronizer(synchronizerId);
+
+  // Add viewports to VOI synchronizers
+  [viewportId1, viewportId2, viewportId3].forEach((viewportId) => {
+    synchronizer.add({
+      renderingEngineId,
+      viewportId,
+    });
+  });
+  synchronizer.setEnabled(false);
```

Comment:
please add a comment that this is just for the purpose of this demo, in reality you want it on